tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import re
+
+import pytest
+import random
+from tests.helpers import get_worker_nodes
+from ocs_ci.framework.pytest_customization.marks import tier4a
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required
+)
+from ocs_ci.ocs import (
+    machine, constants, ocp, node
+)
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@ignore_leftovers
+@aws_platform_required
+@ipi_deployment_required
+class TestNodeReplacement(ManageTest):
```

Comment:
Code looks good but I don't see even one assert statement so I wonder where are we verifying things are working as expected

Reply:
All the function used are having the assert added so i feel no any addition assert is required